test/arg_max_min_test.js

```diff
@@ -0,0 +1,620 @@
+          axes: [1],
+          keepDimensions: true,
+        });
+  });
+
+  it('argMin axes=[1] both keepDimensions and selectLastIndex being true', function() {
+    testArgMaxMin(
+        {
+          shape: [3, 3],
+          value: [
+            1, 2, 3,
+            3, 0, 0,
+            2, 5, 2,
+          ],
+        },
+        {
+          shape: [3, 1],
+          value: [0, 2, 2],
+        },
+        argMin,
+        {
+          axes: [1],
+          keepDimensions: true,
+          selectLastIndex: true,
+        });
+  });
+
+  it('argMin 3D of shape [1, 3, 4] axes=[1] keepDimensions=false selectLastIndex=true', function() {
+    testArgMaxMin(
+        {
+          shape: [1, 3, 4],
+          value: [
+            1, 2, 1, 0,
+            3, 4, 2, 2,
+            5, 0, 5, 0,
+          ],
+        },
+        {
+          shape: [1, 4],
+          value: [0, 2, 0, 2],
+        },
+        argMin,
+        {
+          axes: [1],
+          keepDimensions: false,
+          selectLastIndex: true,
+        });
+  });
+
+  it('argMin 3D discontinuous axes=[0, 2]', function() {
+    testArgMaxMin(
+        {
+          shape: [2, 3, 4],
+          value: [
+            1, 2, 3, 3,
+            3, 4, 2, 9,
+            5, 9, 5, 0,
+            8, 7, 1, 6,
+            9, 3, 5, 2,
+            1, 9, 0, 4,
+          ],
+        },
+        {
+          shape: [3],
+          value: [0, 2, 3],
+        },
+        argMin,
+        {
+          axes: [0, 2],
+        });
+  });
+
+  it('argMin 3D discontinuous axes=[2, 0] selectLastIndex=true', function() {
+    testArgMaxMin(
+        {
+          shape: [2, 3, 4],
+          value: [
+            1, 2, 3, 3,
+            3, 4, 2, 9,
+            5, 9, 5, 0,
+            8, 7, 1, 6,
+            9, 3, 5, 2,
+            1, 9, 0, 4,
+          ],
+        },
+        {
+          shape: [3],
+          value: [6, 7, 6],
+        },
+        argMin,
+        {
+          axes: [2, 0],
+          selectLastIndex: true,
+        });
+  });
+
+  it('argMin 3D axes=[1, 2]', function() {
+    testArgMaxMin(
+        {
+          shape: [2, 3, 4],
+          value: [
+            1, 2, 1, 3,
+            3, 4, 2, 2,
+            5, 0, 5, 0,
+            8, 7, 5, 5,
+            2, 4, 9, 2,
+            1, 2, 4, 1,
+          ],
+        },
+        {
+          shape: [2],
+          value: [9, 8],
+        },
+        argMin,
+        {
+          axes: [1, 2],
+        });
+  });
+
+  it('argMin 3D axes=[2, 1] selectLastIndex=true', function() {
+    testArgMaxMin(
+        {
+          shape: [2, 3, 4],
+          value: [
+            1, 2, 1, 3,
+            3, 4, 2, 2,
+            5, 0, 5, 0,
+            8, 7, 5, 5,
+            2, 4, 9, 2,
+            1, 2, 4, 1,
+          ],
+        },
+        {
+          shape: [2],
+          value: [11, 11],
+        },
+        argMin,
+        {
+          axes: [2, 1],
+          selectLastIndex: true,
+        });
+  });
+});
```

Comment:
For completeness, we should include an empty axes case:

```
  it('argMin axes=[]', function() {
    testArgMaxMin(
        {
          shape: [3, 3],
          value: [
            1, 2, 3,
            3, 0, 0,
            2, 5, 2,
          ],
        },
        {
          shape: [3, 3],
          value: [
            0, 0, 0,
            0, 0, 0,
            0, 0, 0,
          ],
        },
        argMin,
        {
          axes: [],
        });
  });
```

And the corner case for scalar (which can only ever return a scalar of 0):

```
  it('argMin scalar axes=[]', function() {
    testArgMaxMin(
        {
          shape: [],
          value: [3],
        },
        {
          shape: [],
          value: [0],
        },
        argMin,
        {
          axes: [],
        });
  });
```

(note in these cases, `keepDimensions` and `selectLastIndex` have no effect on the output)